app/src/main/res/values/strings.xml

```diff
@@ -62,6 +62,7 @@
     <string name="sender_only">Sender only</string>
     <string name="enable_delivery_reports">Enable delivery reports</string>
     <string name="use_simple_characters">Remove accents and diacritics at sending messages</string>
+    <string name="send_on_enter">Send when enter key is pressed</string>
```

Comment:
change this to "Send message on pressing Enter" and add to all language files please, obviously in english